Tools/CISupport/ews-build/steps.py

```diff
@@ -4375,7 +4375,7 @@ def run(self):
 
         rc = yield super(RunAPITests, self).run()
 
-        if rc == SUCCESS:
+        if rc in [SUCCESS, WARNINGS] and 'without-change' not in self.name:
```

Comment:
Can change the 'without-change' check to be more specific by matching to exact step name, e.g.:
`self.name != RunAPITestsWithoutChange.name`